docs/sidebar.json

```
@@ -66,6 +66,7 @@
       "ecosystem/logging",
       "ecosystem/contributing",
       "ecosystem/sqa",
+      "ecosystem/bugbounty",
```

@vinckr I think this breaks Kratos docs bulds:
```json
Unable to build website for locale "en".
Error: Server responded to https://raw.githubusercontent.com/ory/docs/master/docs/docs/ecosystem/bugbounty.md with status code 404:
404: Not Found
```

---

True, the sidebar has the wrong ID. It should be
```suggestion
      "ecosystem/security",
```